### ISSUE TITLE
Lab seems unreasonable

### ISSUE THREAD

**Issue by howardemilya** [learn-co-curriculum/js-templates-advanced-templating-lab#10]
The instructional READMEs prior to this lab are extremely hard to follow. This lab seems to be built for someone with MUCH more knowledge than what has been taught so far in the curriculum and made me feel like I either missed 10-20 lessons or am extremely stupid. There was no chance I could have come up with the solution to this lab by myself (or even with googling for help). Without the solution available (which I still do not understand), I would not have "passed" this lab.

**Comment by drakeltheryuujin** [comment]
Hi @howardemilya,
We’ve built your feedback into our next major release of this section of curriculum. We’re targeting delivery in the first half of 2019.
